binsync/ui/panel_tabs/util_panel.py

```diff
@@ -294,3 +324,40 @@ def _handle_auto_pull_toggle(self, state):
         else:
             self.controller.auto_pull_enabled = True
 
+class ClientWorker(QObject):
+    finished = pyqtSignal()
+    def __init__(self):
+        super().__init__()
+        self.connected = False
+        
+    def run(self):
+        import urllib.parse
```

Comment:
Never import outside of the first lines of the Python file unless you have a _really_ good reason. In this case, you do not.